packages/block-editor/README.md

```diff
@@ -352,7 +352,7 @@ Undocumented declaration.
 <a name="SETTINGS_DEFAULTS" href="#SETTINGS_DEFAULTS">#</a> **SETTINGS_DEFAULTS**
 
 The default editor settings
-
+```
```

Comment:
Any reason for this wrappers? I don't recall us doing the same in other JSDocs

Reply:
well the markdown looked badly without them

Reply:
cc @nosolosw any impact on the generated docs if we use this kind of notations. I personally don't have strong opinions on the formats... but I like consistency, I don't think this should be a one of change.

Reply:
I see those are added manually: every time this doc is regenerated, it's going to be overwritten.

Reply:
Going to try one thing real quick and commit to this branch.

Reply:
Ohh right I didn't even notice, this should be added here instead https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/store/defaults.js

Reply:
So, two ideas:

- I think we want to add support for the `@property` tag in docgen so this is nicely generated as we want https://github.com/WordPress/gutenberg/issues/15186
- One shortcut while we don't have that is to edit the JSDoc comment (what Riad shared) to something different with a better format. For example: 
```
  - alignWide (`boolean`): Enable/Disable Wide/Full Alignments
  - availableLegacyWidgets (`Array`): Array of objects ...
```

docgen would recognize that markdown and would style accordingly.

Reply:
Thanks @nosolosw 

Let's consider these improvements separately, this is ready to land.

Reply:
Also: we could also add the backticks in the JSDoc comment itself. It'll have the exact same look Andrei suggested.